packages/ast-spec/src/declaration/ImportDeclaration/spec.ts

```diff
@@ -41,5 +41,6 @@ export interface ImportDeclaration extends BaseNode {
    * import 'mod';
    * ```
    */
+  // TODO: we should have a way to differentiate between these two cases
   specifiers: ImportClause[];
```

Comment:
For the future: `specifiers` should be able to be `null`, like how `typeParameters` can be both empty or `null`

Reply:
This won't ever happen, sadly. The ESTree spec is pretty well locked in once defined and so we're stuck with the decisions of the past.

If we change something like this then we're liable to break all sorts of rules that assume it's always defined.

In general we can only ever change our AST extension shapes and the "core" AST is fixed.

Reply:
Hhhh, I forgot this is not TS specific. Fine then.